proj/sub2/bin/foo_main.ml

```diff
@@ -11,42 +11,32 @@ open Printf
    You may want to use Arg.parse_argv to read the remaining
    command-line arguments.
 *)
-let run _argv_offset =
-  Proj_sub2.A.do_something ()
+let run _argv_offset = Proj_sub2.A.do_something ()
 
-let walk _argv_offset =
-  print_endline "Nice."
+let walk _argv_offset = print_endline "Nice."
 
 (* Add your own subcommands as needed. *)
-let subcommands = [
-  "run", run;
-  "walk", walk;
-]
+let subcommands = [ ("run", run); ("walk", walk) ]
 
 let help () =
   let subcommand_names =
     String.concat "\n" (List.map (fun (name, _f) -> "  " ^ name) subcommands)
   in
-  let usage_msg = sprintf "\
-Usage: %s SUBCOMMAND [ARGS]
-where SUBCOMMAND is one of:
-%s
-
-For help on a specific subcommand, try:
-  %s SUBCOMMAND --help
-"
-    Sys.argv.(0) subcommand_names Sys.argv.(0)
+  let usage_msg =
+    sprintf
+      "Usage: %s SUBCOMMAND [ARGS]\n\
+       where SUBCOMMAND is one of:\n\
+       %s\n\n\
+       For help on a specific subcommand, try:\n\
+      \  %s SUBCOMMAND --help\n"
```

Comment:
What's happening here with the leading backslash?

Reply:
I'll let you take a look in case it's a bug with ocamlformat that needs reporting or something.

Reply:
You know what, nevermind. I'll just fix it.

Reply:
Sorry I didn't notice this 😅, Thanks for fixing this!